src/ReflectionType.php

```diff
@@ -85,7 +85,7 @@ public static function convertToDisplayType(\ReflectionType $type)
             'int'  => 'integer',
             'bool' => 'boolean'
         ];
-        $displayType = $type->type;
+        $displayType = (string) $type;
```

Comment:
Please locate other type cast operators in the code (not added by you) and see which style is used most. Then use most used style.